### ISSUE TITLE
[Bug] Integration breaks on upgrade to HA 2023-03

### ISSUE THREAD

**Issue by dai80027** [JoshuaMulliken/ha-wyzeapi#463]
**Describe the bug**
Integration breaks on upgrade to HA 2023-03

**To Reproduce**

Upgrade to 2023-03

**Expected behavior**
Integration still works

**System configuration**
System: rPi 3
HA Version: 2023-03
WyzeApi Version: 0.5.16

**home-assistant.log**

```
Logger: homeassistant.util.package
Source: util/package.py:107
First occurred: 3:30:48 PM (3 occurrences)
Last logged: 3:31:23 PM

Unable to install package wyzeapy==0.5.16: ERROR: Cannot install wyzeapy==0.5.16 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip is available: 23.0 -> 23.0.1 [notice] To update, run: pip install --upgrade pip```

**Comment by brg468** [comment]
Duplicate of https://github.com/JoshuaMulliken/ha-wyzeapi/issues/462. Temporary fix in the comments, permanent fix waiting to be merged.

**Comment by dai80027** [comment]
Thanks!

**Comment by JoeSchubert** [comment]
Can we get confirmations of if this is corrected in version 0.1.19

**Comment by brg468** [comment]
0.1.19 confirmed working on the latest HA dev branch
